runner/src/check_todo.rs

```diff
@@ -15,16 +15,48 @@
 //
 
 use super::*;
+use regex::Regex;
+use std::cmp::max;
 
 /// A [`Runnable`] command that checks for the existence of todos in the codebase with no associated
 /// GitHub issue number.
 pub struct CheckTodo {
     path: String,
+    todo_pattern: Regex,
 }
 
 impl CheckTodo {
     pub fn new(path: String) -> Box<Self> {
-        Box::new(CheckTodo { path })
+        Box::new(CheckTodo {
+            path,
+            todo_pattern: Regex::new(r"TODO\(#\d+\)").expect("Could not parse regex"),
```

Comment:
I think this is re compiling the same regex for each instance of this struct. It seems more efficient to create the regex just once with once_cell or lazy_static instead.

Reply:
Sorry I didn't mean a literal `once_cell` (which as you figured works but is not very convenient to use for a lazy initialization), rather something like

https://github.com/project-oak/oak/blob/3247b75d335729467ccca375e95f15590c8030e0/runner/src/main.rs#L63

Reply:
Thanks. Fixed.